.circleci/config.yml

```diff
@@ -15,7 +15,7 @@ experimental:
 templates:
   job_template: &job_template
     docker:
-      - image: datadog/datadog-agent-runner-circle:go1189
+      - image: datadog/datadog-agent-runner-circle:go1189-with-java
```

Comment:
should we keep that after the PR is merged? should we revert that?

Reply:
We can keep like this, this is another team that manage the build of circle-ci image and they gave me this tag.
The PR (with java for circle-ci) is already merged, they plan to build another image go1190

Reply:
I added a comment with the related PR